model/src/main/java/com/epam/esm/entity/GiftCertificateQueryParameters.java

```diff
@@ -0,0 +1,151 @@
+package com.epam.esm.entity;
+
+/**
+ * The type GiftCertificateQueryParameters.
+ *
+ * @author Verbovskiy Sergei
+ * @version 1.0
+ */
+public class GiftCertificateQueryParameters implements BaseEntity {
```

Comment:
This class is not the Entity class, entity classes are usually represent db tables, so extending from BaseEntity is incorrect

Reply:
> This class is not the Entity class, entity classes are usually represent db tables, so extending from BaseEntity is incorrect

Thank you, fixed